livekit-plugins/livekit-plugins-deepgram/livekit/plugins/deepgram/stt.py

```diff
@@ -391,6 +401,12 @@ def _process_stream_event(self, data: dict) -> None:
 
         elif data["type"] == "Metadata":
             pass  # metadata is too noisy
+        elif data["type"] == "UtteranceEnd":
```

Comment:
Is it being sent each time we finished speaking? If so we should remove the previous dispatch of END_OF_SPEECH to avoid duplicated events. (only when end_of_utterance_ms is enabled)